1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/ModuleManager.kt

```diff
@@ -36,157 +29,12 @@ object ModuleManager : Listenable {
-                ConsoleSpammer::class.java,
-                NoScoreboard::class.java,
-                LongJump::class.java,
-                Parkour::class.java,
-                LadderJump::class.java,
-                FastBow::class.java,
-                MultiActions::class.java,
-                AirJump::class.java,
-                AutoClicker::class.java,
-                NoBob::class.java,
-                BlockOverlay::class.java,
-                NoFriends::class.java,
-                BlockESP::class.java,
-                Chams::class.java,
-                Clip::class.java,
-                Phase::class.java,
-                ServerCrasher::class.java,
-                NoFOV::class.java,
-                FastStairs::class.java,
-                SwingAnimation::class.java,
-                Derp::class.java,
-                ReverseStep::class.java,
-                TNTBlock::class.java,
-                InventoryCleaner::class.java,
-                TrueSight::class.java,
-                LiquidChat::class.java,
-                AntiBlind::class.java,
-                NoSwing::class.java,
-                BedGodMode::class.java,
-                BugUp::class.java,
-                Breadcrumbs::class.java,
-                AbortBreaking::class.java,
-                PotionSaver::class.java,
-                CameraClip::class.java,
-                WaterSpeed::class.java,
-                Ignite::class.java,
-                SlimeJump::class.java,
-                MoreCarry::class.java,
-                NoPitchLimit::class.java,
-                Kick::class.java,
-                Liquids::class.java,
-                AtAllProvider::class.java,
-                AirLadder::class.java,
-                GodMode::class.java,
-                TeleportHit::class.java,
-                ForceUnicodeChat::class.java,
-                ItemTeleport::class.java,
-                BufferSpeed::class.java,
-                SuperKnockback::class.java,
-                ProphuntESP::class.java,
-                AutoFish::class.java,
-                Damage::class.java,
-                Freeze::class.java,
-                KeepContainer::class.java,
-                VehicleOneHit::class.java,
-                Reach::class.java,
-                HeadRotations::class.java,
-                NoJumpDelay::class.java,
-                BlockWalk::class.java,
-                AntiAFK::class.java,
-                PerfectHorseJump::class.java,
-                HUD::class.java,
-                TNTESP::class.java,
-                ComponentOnHover::class.java,
-                KeepAlive::class.java,
-                ResourcePackSpoof::class.java,
-                NoSlowBreak::class.java
-        )
-
+        for (aClass in Reflections(ModuleManager::class.java.getPackage().toString().replace("package ", "")).getTypesAnnotatedWith(ModuleInfo::class.java)) {
+            registerModule(aClass.newInstance() as Module)
+        }
```

Comment:
Kann Probleme beim Remapping geben, addet Module, die Experimental seien sollen, muss man gucken, ob man das will

Reply:
Ich hatte bisher auch bei persönlichen projecten keine Probleme damit. Damit experimentelle Module nicht geadded werden könnte ich einen boolean in moduleinfo einbauen